bump_version

```diff
@@ -18,22 +17,17 @@ if [[ -z $1 ]]; then
     exit 1
 fi
 
-shared_version_file="src/components/apphost.js"
 build_file="./build.yaml"
+package_file="./package*.json"
 
 new_version="$1"
 
-# Parse the version from shared version file
-old_version="$( grep "appVersion" ${shared_version_file} | head -1 | sed -E "s/var appVersion = '([0-9\.]+)';/\1/" | tr -d '[:space:]' )"
-echo "Old version in appHost is: $old_version"
-
-# Set the shared version to the specified new_version
-old_version_sed="$( sed 's/\./\\./g' <<<"${old_version}" )" # Escape the '.' chars
-new_version_sed="$( cut -f1 -d'-' <<<"${new_version}" )"
-sed -i "s/${old_version_sed}/${new_version_sed}/g" ${shared_version_file}
-
+# Parse the version from build.yaml
 old_version="$( grep "version:" ${build_file} | sed -E 's/version: "([0-9\.]+[-a-z0-9]*)"/\1/' )"
-echo "Old version in ${build_file}: ${old_version}"
+echo "Old version: ${old_version}"
+
+# Bump the NPM version
+npm --no-git-tag-version version v${new_version}
```

Comment:
Don't we need to remove `-*` suffix from the version passed to NPM?

Reply:
There isn't that suffix in the `build.yaml` that this reads the version from, so no need.

Reply:
`new_version` is the input param and https://github.com/jellyfin/jellyfin-web/blob/36f195e643534ef46115c64b7796427cebbd0b09/bump_version#L36-L38